src/app/containers/Ad/Amp/AdSlot.test.jsx

```diff
@@ -15,3 +15,31 @@ describe('Amp AdSlot', () => {
     );
   });
 });
+
+describe('getDataSlot', () => {
```

Comment:
Because we are manipulating process.env. SIMORGH_APP_ENV in this test. 

I looked at other examples of where we do this in our code base and we add this inside the describe block:
```
  afterAll(() => {
    delete process.env.SIMORGH_APP_ENV;
  });
```

Reply:
Oh yes, I'll add that block. Thanks